### ISSUE TITLE
Start using git flow

### ISSUE THREAD

**Issue by ozeranskiy** [sunscrapers/djoser#408]
https://danielkummer.github.io/git-flow-cheatsheet/index.html

**Comment by hvdklauw** [comment]
Seems a bit strange to make an issue about.
Git flow is _a_ way to work with git, but not always the best fit.

**Comment by ozeranskiy** [comment]
methodology is an architectural question.
I suggest using, for example, gitflow. This will make it possible to conveniently manage new features and hotfixes.

**Comment by benzkji** [comment]
yes, at least a default "develop" branch, that clearly states "development is going on here", and a master one, with only stable and tagged releases would be cool.

I personally use git flow for my OS projects...it might look like an overkill for small projects, but really helps you to keep things as they should be ;-)
